docs/gettingstarted/dev-tools.rst

```diff
@@ -0,0 +1,29 @@
+Debug tools
+===========
+
+We have built in support for some debug tools, turned off by default.
+You can control them with environmental variables:
+
+``ENABLE_DEBUG_TOOLBAR``
+  Controls `django-debug-toolbar <https://github.com/jazzband/django-debug-toolbar>`. Defaults to ``False``
+
+``ENABLE_SILK``
+  Controls `django-silk <https://github.com/jazzband/django-silk>`. Defaults to ``False``
+
+
+Enabling Debug Tool (example)
+-----------------------------
+#. Set environment variable.
+
+   .. code-block:: console
+
+    $ export ENABLE_SILK='True'
+
+#. Install packages from requirements_dev.txt:
+
+   .. code-block:: powershell
+
+    python -m pip install -r requirements_dev.txt
```

Comment:
Why `python -m`? I guess `pip install -r requirements_dev.txt` is clear and also consistent with rest of the docs.